src/modules/imageresizer/ui/Views/InputPage.xaml

```diff
@@ -121,22 +147,22 @@
                                    TextAlignment="Center"
                                    Visibility="{Binding ShowHeight,Converter={StaticResource BoolValueConverter}}"/>
 
-                        <TextBox Width="56"
-                                 Style="{StaticResource DisabledWhenUnselectedTextBoxStyle}"
-                                 TextWrapping="Wrap"
-                                 AutomationProperties.Name="{x:Static p:Resources.Height}"
-                                 Visibility="{Binding ShowHeight,Converter={StaticResource BoolValueConverter}}">
-                            <TextBox.Text>
-                                <Binding Converter="{StaticResource AutoDoubleConverter}"
-                                         Path="Height"
-                                         UpdateSourceTrigger="PropertyChanged">
-                                    <Binding.ValidationRules>
-                                        <local:AutoDoubleValidationRule/>
-                                    </Binding.ValidationRules>
-                                </Binding>
-                            </TextBox.Text>
-                        </TextBox>
-                        <ComboBox Margin="8,0,0,0"
+                            <ui:NumberBox Style="{StaticResource DisabledWhenUnselectedTextBoxStyle}"
+                                          SpinButtonPlacementMode="Compact"
+                                          Width="102"
+                                          AutomationProperties.Name="{x:Static p:Resources.Height}"
+                                          Visibility="{Binding ShowHeight,Converter={StaticResource BoolValueConverter}}">
+                                <ui:NumberBox.Text>
+                                    <Binding Converter="{StaticResource AutoDoubleConverter}"
```

Comment:
Since this control is changed from `TextBox` to `NumberBox`, the `AutoDoubleConverter` is not the correct converter since it converts a double to a string.